### DISCUSSION TITLE
Idea: running app in silent mode

### DISCUSSION THREAD

**Discussion by Eugenii10** [geocine/remeta#5] [Ideas]
Hello!
Very nice app! Can we have a command argument that will tell to the app to execute in silent mode without confirmation of successful removal metadata from file? E.g. `remeta.exe /s remove 1.png`.
It will be very useful for batch execution on tens and hundreds of files.
Thanks!

**Comment by geocine** [comment]
@Eugenii10  This is implemented on https://github.com/geocine/remeta/releases/tag/v1.1  you can use the `-s` or `--silent` option
```
remeta -s remove <image>
```
